composer.json

```diff
@@ -31,7 +32,7 @@
         "symfony/css-selector": ">=2.7 <4.0",
         "symfony/dom-crawler": ">=2.7.5 <4.0",
         "behat/gherkin": "~4.4.0",
-        "sebastian/comparator": "~1.1",
+        "sebastian/comparator": ">1.1 <1.2 || ^2.0",
```

Comment:
I was just being cautious, as in the other ones. Please let me know what range would work better.